src/shared/awswrangler_functions.py

```diff
@@ -35,3 +38,25 @@ def get_s3_study_meta_list(
         ),
         suffix=extension,
     )
+
+
+def generate_csv_from_parquet(
+    bucket_name: str, bucket_root: str, subbucket_path: str, to_path: str | None = None
+):
+    """Convenience function for generating csvs for dashboard upload
+
+    TODO: Remove on dashboard parquet/API support"""
+    if to_path is None:
+        to_path = f"s3://{bucket_name}/{bucket_root}/{subbucket_path}".replace(".parquet", ".csv")
+    last_valid_df = awswrangler.s3.read_parquet(
+        f"s3://{bucket_name}/{bucket_root}" f"/{subbucket_path}"
+    )
+    last_valid_df = last_valid_df.apply(lambda x: x.strip() if isinstance(x, str) else x).replace(
+        '""', numpy.nan
+    )
+    awswrangler.s3.to_csv(
+        last_valid_df,
+        to_path,
+        index=False,
+        quoting=csv.QUOTE_MINIMAL,
```

Comment:
Minimal is my own preferred dialect, but if this csv file is headed towards the dashboard, the way Vlad has talked about quoting makes me think he expects CSVs to be using the `csv.QUOTE_STRINGS` dialect (or at least `csv.QUOTE_NONNUMERIC` if you don't have Python 3.12).

Reply:
I swear we had a contract at one point indicating this format was ok.

Reply:
Maybe you do? I just said that based on how Vlad keeps talking about quotes casually in convos, but maybe in some spots this is the right quoting.